tools/run_tests/performance/config.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2016, Google Inc.
```

Comment:
I think config.py might be a bit to general name for a module, but I can live with that if it speeds things up.